### PR TITLE
Python 3: Port tests in fetch/api [part 2]

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

**Comment by ziransun** [comment]
Together with changes at [PR 24617](https://github.com/web-platform-tests/wpt/pull/24617) and [PR 24451 ](https://github.com/web-platform-tests/wpt/pull/24451), this should complete all handler related test porting in fetch/api

File: fetch/api/resources/bad-chunk-encoding.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import time
 
+from six.moves import xrange
```
Comment (annevk) [inline fetch/api/resources/bad-chunk-encoding.py:3]: Let's use range instead if that works across both.

**Review by annevk — APPROVED** [review]
Thanks, just one tiny nit.

**Comment by annevk** [comment]
Wait, why do we need to import range?

**Comment by ziransun** [comment]
> Wait, why do we need to import range?

 'xrange' is not defined in Python 3, it's renamed as  'range'  If we can change original 'xrange' to 'range' in the line, I think we can remove the six import line.

**Review by annevk — APPROVED** [review]
Yeah, that's what I meant. Thanks!

**Review by foolip — APPROVED** [review]

**Pull request merged**